### PR TITLE
LPS-118181 The default user icon is not the same in the sidebar Info panel in Content Dashboard as in the Personal Menu

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**

When displaying the content dashboard info panel, the avatar is different from the avatar in the personal menu or the avatar in each row. We want all of them to be the same.

**Evidence**

LPS-118181-gif

### CODE REVIEW

**Comment by dgarciasarai** [comment]
Hey @cgoncas @darquesdev, can you review the backend part? (https://github.com/liferay-tango/liferay-portal/pull/95/commits/c67550abf46d1a5be75cf13edf395546c8cac33f) Once you review it, I will send this pull request to frontend infra to validate the frontend part. Thanks!

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
Hey @cgoncas @darquesdev, can you give me feedback for this commit https://github.com/liferay-tango/liferay-portal/pull/95/commits/188ed1526c57c00dafdd0613caa6f9cdfa863bba? It's the only one changing backend code. Thanks!!

**Comment by darquesdev** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/JournalArticleContentDashboardItem.java [hunk 1]
```diff
@@ -273,14 +273,18 @@ public String getUserPortraitURL(HttpServletRequest httpServletRequest) {
...
 
-		try {
-			return _user.getPortraitURL(themeDisplay);
-		}
-		catch (PortalException portalException) {
-			_log.error(portalException, portalException);
+		String userPortraitURL = StringPool.BLANK;
 
-			return StringPool.BLANK;
+		if (_user.getPortraitId() > 0) {
```
Comment (darquesdev) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/JournalArticleContentDashboardItem.java:278]: It's ok, actually there are blocks of code in Portal like this, but why didn't you check the condition at the beginning? this way we may save variable declaration:

```java

@Override
	public String getUserPortraitURL(HttpServletRequest httpServletRequest) {

		if (_user.getPortraitId() <= 0) {
			return StringPool.BLANK;
		}
		
		ThemeDisplay themeDisplay =
			(ThemeDisplay)httpServletRequest.getAttribute(
				WebKeys.THEME_DISPLAY);

		try {
			return _user.getPortraitURL(themeDisplay);
		}
		catch (PortalException portalException) {
			_log.error(portalException, portalException);
			return StringPool.BLANK;
		}
	}
```

Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/item/JournalArticleContentDashboardItem.java:278]: Right! I'll change it! 👍

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:forward

**Pull request closed without merging**